tests/test_pdf2txt.py

```diff
@@ -1,21 +1,27 @@
 import unittest
 import os
+import sys
+import io
 
 from tools.pdf2txt import (
     handle_input_variables,
     convert_from_pdf,
     ConverterParams,
     OutputType,
     split_by_chapters,
-    write_chapters_to_files
+    write_chapters_to_files,
+    _print_help_message
     )
 from pdfminer.layout import LAParams
 
 
 class TestPdf2Text(unittest.TestCase):
 
+    def setUp(self):
+        self.path_simple1_pdf = [os.path.join(os.path.dirname(__file__), '../samples/simple1.pdf')]
+        ConverterParams.get_chapters = False
```

Comment:
Use an instance of ConverterParams instead of modifying the class's member variables. This is really dangerous because other tests may depend on the default values and if they are changed then this will pollute them.

e.g.:
```
converter_params = ConverterParams(pagenos=set(), laparams=LAParams())
converter_params.get_chapters = False
```

Reply:
Agreed. It is fixed.